mobile-app/api-client/LocalplayService.ts

```diff
@@ -0,0 +1,38 @@
+import { GameState, Coordinates } from "../utils";
```

Comment:
To be consistant we need to use lowerCamelCase for this file(and UpperCamelCase for react components)